app/src/main/java/io/apicurio/registry/services/tenant/CustomTenantConfigResolver.java

```diff
@@ -16,41 +16,55 @@
 
 package io.apicurio.registry.services.tenant;
 
+import io.apicurio.registry.mt.TenantContext;
 import io.apicurio.registry.mt.metadata.TenantMetadataDto;
-import io.apicurio.registry.rest.Headers;
 import io.apicurio.registry.storage.RegistryStorage;
 import io.apicurio.registry.types.Current;
 import io.quarkus.oidc.OidcTenantConfig;
+import io.quarkus.oidc.OidcTenantConfig.Tls.Verification;
 import io.quarkus.oidc.TenantConfigResolver;
 import io.vertx.ext.web.RoutingContext;
 
+import java.util.Optional;
+
 import javax.enterprise.context.ApplicationScoped;
 import javax.inject.Inject;
 
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+
 @ApplicationScoped
 public class CustomTenantConfigResolver implements TenantConfigResolver {
 
-	@Inject
-	@Current
-	RegistryStorage registryStorage;
+    @Inject
+    @Current
+    RegistryStorage registryStorage;
```

Comment:
It looks like the previous version of this class was using tabs instead of spaces.  @Apicurio/developers - everyone should probably check their IDE settings to make sure that doesn't happen.  Only because when it does, we get diffs like this, which make it hard to see the changes.